demo/demo_primary.py

```diff
@@ -626,8 +626,7 @@ def listen():
       primary_ecu.register_new_secondary, 'register_new_secondary')
 
   server.register_function(
-      primary_ecu.get_last_timeserver_attestation,
```

Comment:
That function is in the reference implementation, which has no concept of XMLRPC and so itself has no issue in Python 2.7.14. The issue is that the demo shouldn't have used it directly. So it doesn't really make sense to add a note to the reference implementation about XMLRPC. But otherwise, I'd agree.